src/sensor/settings.py

```diff
@@ -91,20 +92,29 @@
 # IP when the first request comes in.
 INTERNAL_IPS = ["127.0.0.1"]
 
+ENCRYPT_DATA_FILES = env.bool("ENCRYPT_DATA_FILES", default=True)
+
 # See /env.template
 if not IN_DOCKER or RUNNING_TESTS:
     SECRET_KEY = "!j1&*$wnrkrtc-74cc7_^#n6r3om$6s#!fy=zkd_xp(gkikl+8"
     DEBUG = True
     ALLOWED_HOSTS = []
+    ENCRYPTION_KEY = Fernet.generate_key()
```

Comment:
This is a good point. I made a code change so that encryption settings can be removed from migration_settings.py (6b60b80). For settings.py, I believe this would only be used if running outside of docker or for running tests, so it should be find to have a different encryption key. We could consider removing settings.py in the future or making it more clear the purpose of each settings file and adjusting the settings in those files accordingly.